src/main/kotlin/com/sourcegraph/cody/agent/CodyAgentCodebase.kt

```diff
@@ -39,16 +39,18 @@ class CodyAgentCodebase(val project: Project) {
         val openedFileName = file.name
         val relativeFilePath: String? = ProjectFileUtils.getRelativePathToProjectRoot(project, file)
         ApplicationManager.getApplication().invokeLater {
-          CodyToolWindowContent.getInstance(project)
```

Comment:
The error's stack trace pointed to this place. 

I revamped all `getInstance` into `executeOnInstanceIfNotDisposed`.